docs/content/2.frontier/1.introduction/1.whats-frontier.md

```diff
@@ -1,2 +1,5 @@
 # What is Frontier?
 
+[Frontier](https://github.com/realdecoy/frontier/) is a universal and extensible development toolchain established to expedite product development, management, and post deployment application support. It brings our current domain-level tools under one large umbrella giving the ability to scaffold backend, frontend, QA, mobile and other application tools. 
+
+It offers a set of standard libraries and typing kits that can be use across engineering projects to allow for more consistency software development. With frontier, a set of plugins are available for use by developers which can be used to easily generate base project files contributing to a faster development process.
```

Comment:
I think in this line "... kits that can be use across ...", the "use" should be "used"

Reply:
oh and "consistency" should be "consistent".